relay/sources/relayd/src/api.rs

```diff
@@ -107,45 +103,30 @@ pub fn api(
         },
     );
 
-    //let job_config5 = job_config.clone();
     let shared_files_put = warp::path::peek().and(warp::body::form()).map(
-        move |_peek: warp::filters::path::Peek, simple_map: HashMap<String, String>| {
+        move |peek: warp::filters::path::Peek, simple_map: HashMap<String, String>| {
             let metadata = Metadata::new(simple_map);
-            info!("METADATA : {:?}", metadata.unwrap());
-
-            //metadata_writer(metadata.unwrap(), peek.as_str(), job_config5.clone());
+            metadata_writer(format!("{}", metadata.unwrap()), peek.as_str());
             warp::reply()
         },
     );
 
     let shared_files_head = warp::path::peek()
         .and(warp::filters::query::raw()) // recuperation du parametre ?hash=file-hash
-        .map(|_peek: warp::filters::path::Peek, raw: String| {
-            //let path = parse_path_from_peek(peek);
-
-            let contents = fs::read_to_string("tests/files/metadata.txt")
-                .expect("Something went wrong reading the file");
-
-            let mymeta = Metadata::from_str(&contents);
-
-            info!("{:?}", mymeta);
-
+        .map(|peek: warp::filters::path::Peek, raw: String| {
             warp::reply::with_status(
                 "".to_string(),
-                metadata_hash_checker(
-                    "tests/files/metadata.txt".to_string(),
-                    parse_hash_from_raw(raw),
-                ),
+                metadata_hash_checker(format!("./{}", peek.as_str()), parse_hash_from_raw(raw)),
             )
         });
 
     let routes = warp::get2()
         .and(status.or(stats_simple))
         .or(warp::post2().and(remote_run).and(nodes.or(all).or(node_id)))
         .or(warp::put2().and(shared_files).and(shared_files_put))
-        .or(warp::head().and(shared_files).and(shared_files_head));
+        .or(warp::head().and(shared_files).and(shared_files_head))
+        .with(warp::log("relayd::relay-api"));
 
     let (addr, server) = warp::serve(routes).bind_with_graceful_shutdown(listen, shutdown);
-    info!("Started stats API on {}", addr);
```

Comment:
This one is useful as it gives the information the API is started, and the URL to access it

Reply:
But it's not a stats API anymore, so the message could simply be `Started API on {}`